server/server.js

```diff
@@ -4,35 +4,48 @@ const app = express();
 const port = process.env.PORT || 5000;
 
 const events = [
-
+  {id: 0,
+  name: 'Test Event'},
+  {id: 1,
+    name: 'Test Event'},
+  {id: 2,
+    name: 'Test Event'},
 ];
 
-let idCounter = 0;
+let idCounter = events.length -1;
 
 app.use(bodyParser.json());
 
-app.post('/events', function test2(req,res) {
+app.post('/events', function createEvent(req,res) {
   idCounter += 1;
   const eventObject = {
     id: idCounter,
     name: req.body.name
   };
   events.push(eventObject);
-  console.log(events);
   res.status(201).send({ eventObject });
 });
 
 app.delete('/events/:id',(req, res) => {
   const requestId = events.find((event) => event.id === req.params.id);
-  res.status(202).send(requestId);
-    return;
+  
+  for(let i=0; i<=events.length -1; i += 1){
```

Comment:
It's common practice to return a 404 status code (Not found) when the event is not in the list.

Reply:
I added a 404 status code. If I try to delete a non-existing item status 404 shows up in postman.